frontend/src/utils/generalUtils.ts

```diff
@@ -135,6 +135,35 @@ export const validateSubmitTxInputs = (
   }
 }
 
+export const validateSubmitTxInputsSkyline = (
+  settings: ISettingsState, sourceChain: ChainEnum, destinationChain: ChainEnum,
+  destinationAddr: string, amount: string, bridgeTxFee: string, isNativeToken: boolean
+): string | undefined => {
+  if (BigInt(amount) < BigInt(settings.minValueToBridge)) {
```

Comment:
this if is not needed. For skyline, we dont use settings.minValueToBridge anywhere, only minUtxoValue[someChain]